README.rst

```diff
@@ -6,26 +6,14 @@
     :target: https://circleci.com/gh/dwavesystems/dwave-hybrid
     :alt: Linux/Mac build status
```

Comment:
Include Windows here as well?

Reply:
Will do, thanks. This line is from the appveyor era.